README.md

```diff
@@ -11,6 +12,11 @@ brindando los conociminetos básicos, que posibiliten la comprensión de lo que
 4. [Contadores]()
 5. [Acumuladores]()
 6. [Condionales]()
-7. [Ciclos]()
+7. [Ciclos](ciclo/readme.md)
+
+     Cuando nos encontramos frente al hecho de programar, en ocasiones se presenta la necesidad de ejecutar repetidas veces un conjunto de instrucciones, y el duplicar el codigo,
```

Comment:
Good Job, but I want that you create a new folder and inside build the concept.

Remember using yourself words